ellama.el

```diff
@@ -193,52 +193,43 @@ in.  Default value is (current-buffer).
 	  (llm-chat-prompt-append-response
 	   ellama--chat-prompt prompt)
 	(setq ellama--chat-prompt (llm-make-simple-chat-prompt prompt)))
-      (unwind-protect
-	  (save-excursion
-	    (let* ((start (make-marker))
-		   (end (make-marker))
-		   (window (selected-window))
-		   (insert-text
-		    (lambda (text)
-		      ;; Erase and insert the new text between the marker cons.
-		      (with-current-buffer (marker-buffer start)
-			(let ((pt (point)))
-			  (save-excursion
-			    (goto-char start)
-			    (delete-region start end)
-			    (insert (funcall filter text))
-			    (fill-region start (point)))
-			  (goto-char pt))
-			(when ellama-auto-scroll
-			  (select-window (get-window-with-predicate
-					  (lambda (_)
-					    (eq (current-buffer)
-						(get-buffer buffer))))
-					 t)
-			  (goto-char (point-max))
-			  (recenter -1)
-			  (select-window window))))))
```

Comment:
This is definitely buggy (will focus to the window at the time `ellama-stream` was called). `with-selected-window` should do the correct thing.